### DISCUSSION TITLE
Save & Submit option is not available.

### DISCUSSION THREAD

**Discussion by shwetakondwar** [github/india#828] [Students]
I had previously submitted my project but I got a mail from FUTURE READY TALENT to re-submit it. Now I am not able to re-submit it as "Save and Submit" button/option is not visible. Please help me. @ey-mentor1, @EY-Mentor3, @ey-mentor2 , @EY-Mentor4
![Final 2](https://user-images.githubusercontent.com/97171538/161375530-ae7e1d60-ccad-483e-af74-d623c7b2022d.png)
![Final1](https://user-images.githubusercontent.com/97171538/161375544-a5daf44b-820d-4667-a572-a68900e26f7f.png)

**Comment by ey-mentor1** [comment]
Hello,
The last date for the submission is expired. Due to that reason you are be getting this issue.
Backend team will be sharing the emails to the eligible students as per batch there batch. Once you receive the mail, post that only you can save and submit the project.
Thank you
